assets/app/view/game/buy_sell_shares.rb

```diff
@@ -20,7 +20,7 @@ def render
           .map(&:first).sort.reverse
 
         @pool_shares = @game.share_pool.shares_by_corporation[@corporation].group_by(&:percent).values
-          .map(&:first).sort.reverse
+          .map(&:first).sort_by(&:percent).reverse
```

Comment:
do you need to change the other shares to on line 20?

Reply:
Probably not needed, but I have now followed your suggestion to make it consistent.